src/dataloading/async_transferer.cc

```diff
@@ -72,7 +72,7 @@ TransferId AsyncTransferer::StartTransfer(
     // get tensor information
     t.event.reset(new Event);
     CUDA_CALL(cudaEventCreate(&t.event->id));
-    t.dst.CopyFrom(t.src, stream_);
+    t.dst.CopyFromAsync(t.src, stream_);
```

Comment:
I think this is a legacy file and not used anymore. The corresponding Python code is under `dgl._dataloading`. Maybe we should remove this file in v1.0. cc @nv-dlasalle @mufeili

Reply:
Confirming that AsyncTransferer should be removed.